app/pages/news/tags/index.vue

```diff
@@ -26,32 +26,34 @@
 </template>
 
 <script>
-import {newsBaseURL, tagsURL} from '~/constants'
+import { newsBaseURL, tagsURL } from '~/constants'
 
 export default {
-  data() {
+  data () {
     return {
       recentNews: null,
       allTagsList: []
     }
   },
-  async fetch() {
+  async fetch () {
     this.recentNews = await this.$axios.get(`${newsBaseURL}&per_page=100`)
       .then(res => res.data)
       .catch((error) => {
         console.log(error)
-        this.$nuxt.error({statusCode: 500, message: 'News not found'})
+        this.$sentry.captureException(new Error('New not found ', error))
+        this.$nuxt.error({ statusCode: 500, message: 'News not found' })
       })
     if (this.allTags) {
       this.allTagsList = await this.$axios.get(`${tagsURL}?include=${this.allTags.toString()}&per_page=100`)
         .then(res => res.data)
         .catch((error) => {
           console.log(error)
-          this.$nuxt.error({statusCode: 500, message: 'Tags not found'})
+          this.$sentry.captureException(new Error('Tags not found ', error))
+          this.$nuxt.error({ statusCode: 500, message: 'Tags not found' })
```

Comment:
Do we really need this twice? Should this be `500` or `404`?

Reply:
Generally, I think 500 is the server error (in this case Wordpress API server), and 404 is the error if the route in the site (in this case nuxt) is not existing, or am I wrong? @pvj